src/ActionBar/ActionBar.js

```diff
@@ -55,6 +57,14 @@ const ActionBar = React.forwardRef(({
 
     const HeadingTag = `h${headingLevel}`;
 
+    if (!buttonProps) {
+        buttonProps = {};
+    }
+
+    if (!buttonProps['aria-label']) {
+        buttonProps['aria-label'] = backButtonLabel;
```

Comment:
Similar to the comment below, we shouldn't be assigning things to the props objects passed in, does this do the same thing?
```jsx
                    <Button
                        aria-label={backButtonLabel}
                        {...buttonProps}
                        compact
                        glyph='navigation-left-arrow'
                        onClick={onBackClick}
                        option='transparent' />
```
set the aria-label to backButtonLabel, unless it's defined in buttonProps, then use buttonProps['aria-label']